portfolio/src/main/webapp/script.js

```diff
@@ -12,54 +12,132 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** Adds a random image to the page. */
+
+/** Adds a random picture to the page. */
 function addRandomPic() {
-  const places = ['Alon Hagalil', 'Tel Aviv', 'Mitzpe Ramon', 'Hasbani river', 'Jerusalem', 'Tzipori stream', 'Habonim beach'];
-  // Pick a random place
-  const imageName = places[Math.floor(Math.random() * places.length)];
-  // Add image to the page
-  const imgElement = createImg(imageName);
-  const imageContainer = document.getElementById('image-container');
-  //Add title to image
-  const location = document.getElementById('location-name')  
-  location.innerText = imageName;
-  const titleContainer = document.getElementById('location-title');
-  // Remove the previous image.
-  imageContainer.innerHTML = '';
-  imageContainer.appendChild(imgElement);
-  titleContainer.style.display = "inline-block";
+    const places = ['Alon Hagalil', 'Tel Aviv', 'Mitzpe Ramon', 'Hasbani river', 'Jerusalem', 'Tzipori stream', 'Habonim beach'];  
+    // Pick a random place
+    const imageName = places[Math.floor(Math.random() * places.length)];
+    // Add image to the page
+    const imgElement = createImg(imageName);
+    const imageContainer = document.getElementById('image-container');
+    //Add title to image
+    const location = document.getElementById('location-name')  
+    location.innerText = imageName;
+    const titleContainer = document.getElementById('location-title');
+    document.getElementById('map-title').innerText = "Can you find the right location tag on the map?";
+    // Remove the previous image.
+    imageContainer.innerHTML = '';
+    imageContainer.appendChild(imgElement);
+    titleContainer.style.display = "inline-block";
 }
 
 /** Creates the image element. */
 function createImg(imageName) {
-  const imgUrl = '/images/' + imageName + '.jpg';
-  const imgElement = document.createElement('img');
-  imgElement.src = imgUrl;
-  imgElement.alt = 'I am sorry, the image cannot be displayed';
-  imgElement.width = 300;
-  imgElement.height = 300;
-  return imgElement;
+    const imgUrl = '/images/' + imageName + '.jpg';
+    const imgElement = document.createElement('img');
+    imgElement.src = imgUrl;
+    imgElement.alt = 'I am sorry, the image cannot be displayed';
```

Comment:
What is the event that triggers this?

Reply:
The alt attribute is an alternate text that is displayed by the browser if the image for some reason cannot be displayed (for example error in src/ connection problems). So it is triggered automaticly when there is a problem with the image element.